docs/schedule/examples/continuous-stops.ja.md

```diff
@@ -0,0 +1,71 @@
+---
+search:
+  exclude: true
+---
+
+# 連続した停車
+
+<hr>
+
+## どこでも送迎
+
+交通機関The Current (Rockingham, US-VT) は、ルート2、53、55で、連続停車ポリシーを適用しています。バスが安全に停車できる場所がある限り、乗客はルート上のすべての予定された停留所で送迎を受けることができます。
+
+[routes.txt](../../reference/#routestxt)は、`continuous_pickup`と`continuous_drop_off` というフィールドを使用して、このサービスを記述するためのファイルである。このフィールドは、連続的なピックアップとドロップオフが許可されていることを示すために`0に`設定されています。
+
+[**routes.txt**](../../reference/#routestxt)
+
+    route_id,route_short_name,route_long_name,route_type,continuous_pickup,continuous_drop_off
+    2,2,Bellows Falls In-Town,3,0,0
+    53,53,Bellows Falls / Battleboro Commuter,3,0,0
+    55,55,Bellows Falls / Springfield Shuttle,3,0,0
+
+[ソース例](https://crtransit.org/bus-schedules/)
+
+<hr>
+
+## 路線の一部区間でのピックアップとドロップオフ
+
+ビクターバレー・トランジット（米国カリフォルニア州ビクタービル）は、22号線の一部区間に限り、連続停車制度を適用しています。乗客は、County Fareゾーン内の安全な場所でのみバスに乗り降りすることができます。Local Fareゾーンでは、連続したピックアップとドロップオフはできません。
+
+ローカル料金ゾーンとカウンティ料金ゾーンは、下図のようにエア・エクスプレス・ウェイで区切られています。予定停車駅のNational Trails Highway - Air Expresswayは、この境界のやや北側に位置しています。正確には、交通機関は、バスルートと境界線との実際の交差点に停留所を追加し、そこから連続した送迎を行うことができます。この停留所は予定外のままでもよい。
```

Comment:
I think a couple do not translates got translated here when comparing with other languages